lib/svg-sprite/mode/css/packer.js

```diff
@@ -46,17 +46,17 @@ SVGSpriteCssPacker.prototype = {};
  * @return {Array} shapes       Packed shapes
  */
 SVGSpriteCssPacker.prototype.fit = function() {
-    const { length } = this.blocks;
-    const width = length ? this.blocks[0].width : 0;
-    const height = length ? this.blocks[0].height : 0;
+    if (this.blocks.length === 0) {
+        return this.positions;
```

Comment:
I opted for returning `this.positions`, unsure if the implicit undefined would be OK.

Reply:
this should be
```
this.root.width = 0;
this.root.height = 0;
return this.positions;
```

Reply:
Not sure if that makes sense? I mean, we are returning `this.positions` so `this.root.*` are not used?

Reply:
I mean in original code it sets width and height of 0 to this.root

Reply:
Yeah, but in the original code we didn't return so we were using `this.root` below. Not sure if it's needed now that we are returning early.

Reply:
I'm not sure if this will make sense, but since there is no unit testing we can't be sure about this changes :)

Reply:
@XhmikosR maybe we'll land this PR later after unit testing is done?

Reply:
Sure, although I'm not sure I'll have the time to add a test for this.

Reply:
Should we close it for now?

Reply:
Why close it? It's a perfectly fine change that the current tests don't cover. Worst case, we land it as is and see how it goes.